RELEASE.md

```diff
@@ -8,13 +8,16 @@ To create a changeset run `changeset add` (or  `yarn cs`)
 
 This will bring up an interactive console which asks which packages are affect and if they require minor or major update.
 
-when time to release new versions of npm package run `changeset version` this will look thru the changeset files that have been generated since last release to bump the versions for package automatically to major if any changesets specify major change minor if only minor and patch if a the package had no changesets of its own but depends on one which will be updated.
+## Releasing
 
-finally `changeset publish` will go thru and publish to npm the packages that need publishing.
+This repo is setup to automatically version and release packages
```

Comment:
Nice! I think this explanation is clearer. I accidentally did `changeset publish` the first time

Reply:
well i did say to run that command so thats on me for not giving you the updated way